data/mods/Magiclysm/items/ethereal_items.json

```
@@ -856,5 +856,73 @@
     "relic_data": {
       "passive_effects": [ { "has": "WIELD", "condition": "ALWAYS", "values": [ { "value": "ITEM_DAMAGE_PURE", "add": 10 } ] } ]
     }
+  },
+  {
+    "id": "aura_auroral_guard",
+    "type": "ARMOR",
+    "category": "armor",
+    "name": { "str_sp": "auroral shell" },
+    "description": "A sturdy shell that wraps your body, gaining a much better defense.",
```

```suggestion
    "description": "A sturdy shell that wraps your body, gaining a sturdy defense.",
```
To match the spell description.